jicoco-config/src/main/kotlin/org/jitsi/config/TypesafeConfigSource.kt

```diff
@@ -46,7 +46,23 @@ class TypesafeConfigSource(
             typeOf<Boolean>() -> wrap { key -> config.getBoolean(key) }
             typeOf<Int>() -> wrap { key -> config.getInt(key) }
             typeOf<Long>() -> wrap { key -> config.getLong(key) }
-            typeOf<Double>() -> wrap { key -> config.getDouble(key) }
+            // Support expressions such as "5%"
+            typeOf<Double>() -> wrap { key ->
+                try {
+                    config.getDouble(key)
+                } catch (wrongTypeException: com.typesafe.config.ConfigException.WrongType) {
+                    val stringValue: String = config.getString(key).trim()
+                    if (stringValue.endsWith("%")) {
+                        try {
+                            0.01 * stringValue.dropLast(1).toDouble()
```

Comment:
Any reason why you do `0.01 * <double val>` instead of `<double val> / 100.0`?  Just curious, as the latter feels a bit more obvious.

Reply:
No specific reason, I think it's clear.